apps/exports/tests/test_runner.py

```diff
@@ -31,3 +32,29 @@ def test_custom_server_url(self):
         command = _compile_export_command(export_config, project, force=False)
 
         self.assertIn(server.url, command)
+
+
+class RunnerArgumentsDbTest(TestCase):
```

Comment:
I think this test supercedes the previous one, so you could just remove it... 
Or just change the url of my test to be `settings.COMMCARE_DEFAULT_SERVER` (then we aren't adding an unnecessary db test).

Reply:
see comment below about the rationale for a DB test. though I'm also happy to remove it and add a trailing-slash non-db test if you think that's a more appropriate substitute.